server/src/server.js

```diff
@@ -2,8 +2,8 @@
 const express = require('express');
 const app = express();
 const Ably = require('ably');
-var realtime = new Ably.Realtime({key: "<YOUR-API-KEY>"});
-var rest = new Ably.Rest({key: "<YOUR-API-KEY>"});
+var realtime = new Ably.Realtime({key: "vZ7jrQ.pg_eEg:qVyepOstflFEOuEF"});
```

Comment:
Was adding a real key here intentional @ben-xD?

Reply:
I've just revoked the key on the ably.com dashboard, and updated the code 9a440e2e1aa7708568e856f3b85151a8b4a19a0a